components/ErrorBanner/OldDesignatedVotingAccountWarningBanner.tsx

```diff
@@ -71,7 +71,7 @@ const Link = styled(NextLink)`
   font: inherit;
   color: inherit;
   text-decoration: underline;
-  transition: opacity var(--animation-duration);
```

Comment:
this css var was copied over from somewhere else, but it's not defined in this app.